Parsers/ASimRegistryEvent/Parsers/ASimRegistryEventMicrosoftWindowsEvent.yaml

```diff
@@ -0,0 +1,318 @@
+Parser:
+    Title: Registry Event ASIM parser for Microsoft Windows Events (registry creation event)
+    Version: "0.2.1"
+    LastUpdated: Oct 11th 2023
+Product:
+    Name: Security Events
+Normalization:
+    Schema: RegistryEvent
+    Version: "0.1.0"
+References:
+    - Title: ASIM Registry Schema
+      Link: https://aka.ms/ASimRegistryEventDoc
+    - Title: ASIM
+      Link: https:/aka.ms/AboutASIM
+    - Title: Reference for access rights
+      Link: https://learn.microsoft.com/en-us/windows/win32/sysinfo/registry-key-security-and-access-rights
+Description: |
+    This ASIM parser supports normalizing Microsoft Windows events (event numbers 4657 and 4663), to the ASIM Registry Event normalized schema.
+ParserName: ASimRegistryEventMicrosoftWindowsEvent
+EquivalentBuiltInParser: _ASim_RegistryEvent_MicrosoftWindowsEvent
+ParserParams:
+    - Name: disabled
+      Type: bool
+      Default: false
+ParserQuery: |
+    let parser = (
+    disabled: bool=false
+    ) {
+    let ASIM_GetAccountType = (sid: string) { 
+    iif ( 
+    sid in ("S-1-0-0", "S-1-5-18", "S-1-5-19", "S-1-5-20"),
+    "Simple"
+        ,
+    "Windows"
+    )
+    };
+        let ASIM_ParseWindowsEvents = (WindowsEvent: (EventData: dynamic)) {
+        WindowsEvent
+            | extend
+                ActorUsername = iif(isnotempty(EventData.SubjectDomainName), strcat(EventData.SubjectDomainName, @'\', EventData.SubjectUserName), EventData.SubjectUserName)
+                ,
+                ActorDomainName = tostring(EventData.SubjectDomainName)
+                ,
+                ActorUserId = tostring(EventData.SubjectUserSid)
+                ,
+                ActorSessionId = tostring(EventData.SubjectLogonId)
+                ,
+                ActingProcessName = tostring(EventData.ProcessName)
+                ,
+                ActingProcessId = tostring(toint(tolong(EventData.ProcessId)))
```

Comment:
Can this be formatted ?

Reply:
this output is from post formatting on Log Analytics console